### ISSUE TITLE
options are not considered when rendering chart

### ISSUE THREAD

**Issue by BigMichi1** [symfony/ux#11]
i tried to set some options for the chart in my controller like this

```php
        $chart->setOptions([
            'title' => ['display'=> true, 'text' => $this->translator->trans('headline', ['%year%' => $year])],
            'legend' => ['display' => true, 'position' => 'bottom'],
        ]);
```

this is then rendered as
```json5
{
  // omitting the data part that works
  "options": {
    "title": {
      "display": true,
      "text": "Statistik 2020"
    },
    "legend": {
      "display": true,
      "position": "bottom"
    }
  }
}
```

but the check here https://github.com/symfony/ux/blob/11bde53628b1d66d7b00ce3721cf9e2773e65990/src/Chartjs/Resources/assets/src/controller.js#L18
replaces the options with an empty object, because `payload.options.length` is undefined while `payload.options` contains the options

**Comment by Kocal** [comment]
I think using `typeof payload.options === 'undefined' || Object.keys(payload.options).length === 0` would fix the issue.

**Comment by tgalopin** [comment]
Ah, that's a bug! If you wish to open a PR, go for it! Otherwise I'll create it :)

**Comment by tgalopin** [comment]
Your edit @Kocal was why I did it this way initially. I'm opening a PR :)

**Comment by Kocal** [comment]
Nice :)

**Comment by Gasjki** [comment]
Hey guys! I'm trying to do a workaround until everything is fixed. I extended the Stimulus controller as-in given example, but I don't know how to set the entrypoints. A little more clarity for this as well in documentation?

Thanks!

**Comment by tgalopin** [comment]
Hi @Gasjki!

FYI I implemented a fix, we will release it tomorrow (at the same time as Stimulus 2 support :) ).

Regarding extension, I'm not sure what's your issue. Could you open a dedicated one describing your problem in more details?

**Comment by Gasjki** [comment]
Hi @tgalopin ! My problem is not actually an issue and I think is not necessary to create a new issue here. My question was related to the extensibility. If we extend the JS controller (based on documentation example), how are we supposed to load it? That's not clear for me. 

Thanks!

**Comment by tgalopin** [comment]
This has been fixed and released in 1.1.0, thanks for the report!

@Gasjki for support, please create a new issue or even better, join the Symfony Slack (symfony.com/slack): that's going to be much easier to help you :) !
